api/types/daemon.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "github.com/docker/docker/runconfig"
+
+// ContainerCreateConfig is the parameter set to ContainerCreate()
+type ContainerCreateConfig struct {
```

Comment:
Can we put this in configs.go, or put the content of configs.go in here? I'd prefer to keep all of the "config structs used to talk to the daemon" in one place, whatever the file is called.